### PR TITLE
adding some announcements and logo

### LINKED ISSUE #68: Add Reference and Thanks to AI Verify Foundation and Tensorplex

### PR DESCRIPTION
**Pull request by Bhardwaj-Rishabh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
-# walledeval
+# Walled Eval: Testing LLMs Against Jailbreaks and Unprecedented Harms
 
-> _Test LLMs against jailbreaks and unprecedented harms_
+<p align="center">
+  <img width="700" alt="walleai_logo_shield" src="https://github.com/walledai/walledeval/assets/32847115/d8b1d14f-7071-448b-8997-2eeba4c2c8f6">
+</p>
+
+> 🔥 Excited to announce the release of the community version of our guardrails: [WalledGuard](https://huggingface.co/walledai/walledguard-c)! **Walled Guard** comes in two versions: **Community** and **Advanced***. We are releasing the community version under the Apache-2.0 License. To get access to the advanced version, please contact us at [admin@walled.ai](mailto:admin@walled.ai)
```
Comment (ThePyProgrammer) [inline README.md]: nt: One extra asterisk after "Advanced" messes up the formatting

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+
+> 🔥 Excited to partner with IMDA Singapore-AI Verify to build robust AI safety and controllability measures!
```
Comment (ThePyProgrammer) [inline README.md]: nt: IMDA Singapore-AI Verify -> IMDA Singapore AI Verify Foundation (their full name hasn't been inputted)

File: README.md [hunk 3]
```diff
@@ -1,6 +1,14 @@
-# walledeval
+# Walled Eval: Testing LLMs Against Jailbreaks and Unprecedented Harms
 
-> _Test LLMs against jailbreaks and unprecedented harms_
+<p align="center">
+  <img width="700" alt="walleai_logo_shield" src="https://github.com/walledai/walledeval/assets/32847115/d8b1d14f-7071-448b-8997-2eeba4c2c8f6">
```
Comment (ThePyProgrammer) [inline README.md]: This is way too big, also should either be above the title or at the bottom, not here.

File: README.md [hunk 4]
```diff
@@ -1,6 +1,14 @@
-# walledeval
+# Walled Eval: Testing LLMs Against Jailbreaks and Unprecedented Harms
```
Comment (ThePyProgrammer) [inline README.md]: WalledEval or Walled Eval?

**Review by ThePyProgrammer — CHANGES_REQUESTED** [review]

**Review by Bhardwaj-Rishabh — COMMENTED** [review]
Thanks for pointing out, go ahead with the changes, looks good to me!

**Pull request merged**